core/.changelog.d/4351.added

```diff
@@ -0,0 +1 @@
+Added support for sortedmulti.
```

Comment:
We can also mention [BIP383](https://github.com/bitcoin/bips/blob/master/bip-0383.mediawiki) or [BIP67](https://github.com/bitcoin/bips/blob/master/bip-0067.mediawiki). I'm hesitant to mention descriptors because I think we [do not support](https://github.com/trezor/trezor-firmware/blob/onvej-sl/multisig-fix/core/src/apps/bitcoin/get_public_key.py#L123) multisig descriptors at all. Anyway, "sortedmulti" itself might be somewhat misleading as well.

What about "Added support for lexicographic sorting of pubkeys"?

Reply:
Sounds good, maybe even _... multisig pubkeys_ or _... pubkeys for multisig_?